<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportToDotDialog</class>
 <widget class="QDialog" name="ExportToDotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>182</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExportToDotDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>GST_DEBUG_DUMP_DOT_DIR:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pathLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filenameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="currentBinRadioButton">
       <property name="text">
        <string>Current bin</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mainBinRadioButton">
       <property name="text">
        <string>Main bin (pipeline)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="mediaTypeCheckBox">
       <property name="text">
        <string>Media type</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="capsDetailsCheckBox">
       <property name="text">
        <string>Caps details</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="nonDefaultParametersCheckBox">
       <property name="text">
        <string>Non default parameters</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="statesCheckBox">
       <property name="text">
        <string>States</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportToDotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportToDotDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
